packages/react/tooltip/src/Tooltip.tsx

```diff
@@ -35,18 +35,20 @@ type TooltipProviderContextValue = {
   delayDuration: number;
   onOpen(): void;
   onClose(): void;
+  isPointerInTransitRef: React.MutableRefObject<boolean>;
+  closeOnLeave: boolean;
```

Comment:
Ideas on a better name for this? I'd prefer it to be opt in to avoid something like `PointerAccessibleContent={false}`